packages/debug/src/browser/debug-session.tsx

```diff
@@ -146,11 +146,12 @@ export class DebugSession implements CompositeTreeElement {
                 sourceReference: Number(uri.query)
             };
         }
-        const path = await this.fileSystem.getFsPath(uri.toString());
-        return {
-            name: uri.displayName,
-            path
-        };
+        const name = uri.displayName;
+        let path: string | undefined = uri.toString();
+        if (uri.scheme === 'file') {
+            path = await this.fileSystem.getFsPath(uri.toString());
+        }
```

Comment:
should we assign path to uri.toString() in a else block to avoid to assign it and then override value ?

Reply:
I think it would be good to use `path` as the input in the if statement to avoid calling `toString()` twice.

```typescript
        const name = uri.displayName;
        let path: string | undefined = uri.toString();
        if (uri.scheme === 'file') {
            path = await this.fileSystem.getFsPath(path);
        }
        return { name, path };
```

Reply:
yes probably better

Reply:
thanks, applied